### PR TITLE
Refactor setup commands and remove local setup

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
Remove local setup

### CODE REVIEW

File: components/cli/cli/cli.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ import (
 	"cellery.io/cellery/components/cli/pkg/kubernetes"
 	"cellery.io/cellery/components/cli/pkg/registry"
 	"cellery.io/cellery/components/cli/pkg/registry/credentials"
+	runtime2 "cellery.io/cellery/components/cli/pkg/runtime"
```
Comment (Mirage20) [inline components/cli/cli/cli.go]: Can u rename this to something meaningful like cliruntime?

File: components/cli/cmd/cellery/setup_list_clusters.go [hunk 2]
```diff
@@ -21,16 +21,18 @@ package main
 import (
 	"github.com/spf13/cobra"
 
+	"cellery.io/cellery/components/cli/cli"
+
```
Comment (Mirage20) [inline components/cli/cmd/cellery/setup_list_clusters.go]: Please run code.format

File: components/cli/pkg/util/aws.go [hunk 3]
```diff
@@ -19,100 +19,83 @@
...
-	if err != nil {
-		if aerr, ok := err.(awserr.Error); ok {
-			ExitWithErrorMessage("Error getting size of file", aerr)
-		} else {
-			ExitWithErrorMessage("Error getting size of file", err)
-		}
-	}
-	return *result
-}
+//func DownloadFromS3Bucket(bucket, item, path string, displayProgressBar bool) {
```
Comment (Mirage20) [inline components/cli/pkg/util/aws.go]: Please remove unused files

File: components/cli/cli/cli.go (hunk 1 above)
Comment (MadhukaHarith92) [reply components/cli/cli/cli.go]: Updated.

File: components/cli/pkg/util/aws.go (hunk 3 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/util/aws.go]: Removed.

File: components/cli/cmd/cellery/setup_list_clusters.go (hunk 2 above)
Comment (MadhukaHarith92) [reply components/cli/cmd/cellery/setup_list_clusters.go]: Updated.

**Pull request merged**